test/e2e/features/config.feature

```diff
@@ -97,19 +97,20 @@ Feature: Test configuration settings
         @windows
         Examples:
             | property                        | value1 | value2 |
-            | skip-check-administrator-user   | true   | false  |
             | skip-check-bundle-extracted     | true   | false  |
-            | skip-check-hyperv-installed     | true   | false  |
-            | skip-check-hyperv-switch        | true   | false  |
             | skip-check-user-in-hyperv-group | true   | false  |
-            | skip-check-windows-version      | true   | false  |
+
+    #| skip-check-hyperv-installed     | true   | false  |
+    #| skip-check-hyperv-switch        | true   | false  |
+    #| skip-check-administrator-user   | true   | false  |
+    #| skip-check-windows-version      | true   | false  |
```

Comment:
Why are those commented out?

Reply:
I think those are now handle by the installer, so `setup` does not take care of them. Not sure due to expected roadmap around future on installer if they may go back to `setup` or they should disappear as config options? @anjannath ^^

Reply:
I don't think they'll disappear as config options, these are still present in current `crc`

Only the `fix` part of the preflight checks were moved to the MSI, `crc start` and `crc setup` still runs these checks and having these config options allows users to skip the checks

Reply:
> It will check and...does it fix them in case of fails?

my mistake, only `crc start` runs these preflight checks, `crc setup` doesn't 

> on the other hand skipping them...does not show the skipping message anymore

for `crc start` its shown, but in this case we are only testing `crc setup` with these config options?

```
PS > crc config view
- consent-telemetry                     : no
- skip-check-hyperv-installed           : true
PS > crc start
INFO Checking if running in a shell with administrator rights
INFO Checking Windows release
INFO Checking Windows edition
INFO Checking if Hyper-V is installed and operational
WARN Skipping above check...
INFO Checking if crc-users group exists
INFO Checking if current user is in Hyper-V Admins group
INFO Checking if Hyper-V service is enabled
INFO Checking if vsock is correctly configured
INFO Checking if the daemon task is installed
INFO Checking if the daemon task is running
INFO Checking admin helper service is running
INFO Loading bundle: crc_hyperv_4.11.7_amd64..
[...]
```

Reply:
yeah only on `setup`...so now make sense why they are not showing the skipping message anymore

Reply:
@praveenkumar @adrianriobo just to be clear, this doesn't need any changes for the PR right?